internal/pkg/config/env.go

```diff
@@ -29,18 +29,25 @@ type Environment struct {
 
 // CustomizeEnv represents the custom environment config.
 type CustomizeEnv struct {
-	ImportVPC *ImportVPC `json:"importVPC,omitempty"`
-	VPCConfig *AdjustVPC `json:"adjustVPC,omitempty"`
+	ImportVPC      *ImportVPC `json:"importVPC,omitempty"`
+	VPCConfig      *AdjustVPC `json:"adjustVPC,omitempty"`
+	ImportCertARNs []string   `json:"importCertARNs,omitempty"`
 }
 
-// NewCustomizeEnv returns a new CustomizeEnv struct.
-func NewCustomizeEnv(importVPC *ImportVPC, adjustVPC *AdjustVPC) *CustomizeEnv {
-	if importVPC == nil && adjustVPC == nil {
+// HasImportedCerts return if the environment has imported certs.
+func (e *Environment) HasImportedCerts() bool {
+	return e.CustomConfig != nil && len(e.CustomConfig.ImportCertARNs) != 0
+}
+
+// CustomizeEnvP returns a pointer to the CustomizeEnv value passed in.
+func CustomizeEnvP(input CustomizeEnv) *CustomizeEnv {
```

Comment:
Is this mainly refactoring or is there any other concern that led to this change?

Reply:
This is not a refactoring. This is a shorthand method similar to `stringP` or `aws.String()` that returns nil if it is empty, otherwise return a pointer that points to the struct.

Reply:
So we will have
 
```golang
CustomizeEnv(CustomEnvConfig) *CustomEnvConfig
```

It is a little strange to me 🤔